pom.xml

```diff
@@ -219,17 +224,34 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.springframework.boot</groupId>
-                <artifactId>spring-boot-maven-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
```

Comment:
versions should be moved to the top of the pom file and here should be a link in the form of expression language

Reply:
thank you